src/store/reserved/reservedCharitySlice.js

```diff
@@ -0,0 +1,40 @@
+import { createSlice } from '@reduxjs/toolkit'
+import { getAllReservedCharity } from './reservedThunk'
+
+const initialState = {
+   bookedCharity: [],
+   isLoading: false,
+   error: null,
+}
+
+export const reservedCharitySlice = createSlice({
```

Comment:
Could you please move all functions and all logic from reservedcharity, reservedwish slice and thunk move into only one already exist charity slice and wish slice .